save-cosv/src/main/kotlin/com/saveourtool/save/cosv/controllers/CosvController.kt

```diff
@@ -274,6 +275,31 @@ class CosvController(
                 }
         }
 
+    /**
+     * @param organizationName
+     * @param authentication
+     * @return list of deleted keys
+     */
+    @RequiresAuthorizationSourceHeader
+    @DeleteMapping("/{organizationName}/delete-processed", produces = [MediaType.APPLICATION_JSON_VALUE])
+    fun deleteProcessed(
+        @PathVariable organizationName: String,
+        authentication: Authentication,
+    ): Mono<RawCosvFileDtoList> = hasPermission(authentication, organizationName, Permission.DELETE, "delete")
```

Comment:
Why `Mono<List<RawCosvFileDto>>` and not `Flux<RawCosvFileDto>`?

Reply:
I collect all keys to a single list to make a single call to delete all keys (since it removes keys from database, it's much better),
So I have `Mono<List<RawCosvFileDto>>` after removing, it can be flatten to `Flux<RawCosvFileDto>`, but it's an extra step. Decided to return Mono of List